huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -168,8 +168,13 @@ func ResourceRdsInstance() *schema.Resource {
 						},
 						"keep_days": {
 							Type:     schema.TypeInt,
+							Default:  7,
+							Optional: true,
+						},
+						"period": {
+							Type:     schema.TypeString,
+							Default:  "1,2,3,4,5,6,7",
```

Comment:
it's better to add the default value in the implementation function rather than in the schema.